source/application/BioXAS/BioXASImagingAppController.cpp

```diff
@@ -21,6 +21,7 @@ along with Acquaman.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "BioXASImagingAppController.h"
 
+#include "beamline/CLS/CLSBeamlines.h"
```

Comment:
Yeah, I knew that would be a problem when I suggested it. However, it's a convenience function ... so you don't have to call it. I would say the best thing to do is to return some sort of string that indicates the problem. Where there are problem we can still directly pass a QString as:
`AMRun firstRun("REIXS", CLSBeamline::REIXS);`
I would prefer to stay away from global or namespaced const variables.